server/lib/flow-event.js

```diff
@@ -124,7 +126,7 @@ const metricsRequest = (req, metrics, requestReceivedTime) => {
       }
     }
 
-    amplitude(event, req, metrics);
+    amplitude(event, req, metrics, location);
```

Comment:
same as https://github.com/mozilla/fxa-content-server/pull/6851#discussion_r244338400

Reply:
I was just trying to avoid either cloning or modifying the incoming `metrics` arg is all. I'll clone it instead.

Reply:
Your choice!